test/integration/fixtures/options/bail-with-before.fixture.js

```diff
@@ -37,7 +37,7 @@ describe('suite2', function () {
   before('before suite2', function () {});
   beforeEach('beforeEach suite2', function () {});
   it('test suite2', function () {
-    runOrder.push('test suite2 - should not run');
+    console.log('test suite2 - should not run');
```

Comment:
Shouldn't this be an assertion?

Reply:
describe('suite2', function () {...} ) is skipped by the --bail flag. So this blog does never run.